src/services/getOneRestaurant.js

```diff
@@ -0,0 +1,13 @@
+import { getOneRecord } from '../db/getOneRecord';
+import { createRestaurantPK } from '../libs/createRestaurantPK';
+
+export const getOneRestaurant = async (req, res, next) => {
+    const { name, street, postalCode } = req.query;
+    const pk = createRestaurantPK(name, street, postalCode);
+    try {
+        const result = await getOneRecord(pk, name);
```

Comment:
I would probably do the same here, make `getOneRecord` take one object.

also `pk` is not a very meaningful variable name